src/search-modal/EmptyStates.jsx

```diff
@@ -1,8 +1,30 @@
 /* eslint-disable react/prop-types */
 // @ts-check
 import React from 'react';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { Stack } from '@openedx/paragon';
 import { useStats, useClearRefinements } from 'react-instantsearch';
 
+import EmptySearchImage from './images/empty-search.svg';
+import NoResultImage from './images/no-results.svg';
+import messages from './messages';
+
+const EmptySearch = () => (
+  <Stack className="d-flex mt-6 align-items-center">
+    <p className="fs-large"> <FormattedMessage {...messages.emptySearchTitle} /> </p>
+    <p className="small text-muted"> <FormattedMessage {...messages.emptySearchSubtitle} /> </p>
+    <img src={EmptySearchImage} alt="" />
+  </Stack>
+);
+
+const NoResults = () => (
+  <Stack className="d-flex mt-6 align-items-center">
+    <p className="fs-large"> <FormattedMessage {...messages.noResultsTitle} /> </p>
+    <p className="small text-muted"> <FormattedMessage {...messages.noResultsSubtitle} /> </p>
+    <img src={NoResultImage} alt="" />
+  </Stack>
+);
+
```

Comment:
I think these should be deduplicated since they have the same structure, and will likely need to always have the same structure.

Reply:
Done: 78e0a817637b2b9585b7b4b00d73f9c293a9d90a